main/web_server.c

```diff
@@ -30,6 +32,48 @@ typedef struct web_server_context {
 #define CHECK_FILE_EXTENSION(filename, ext) \
     (strcasecmp(&filename[strlen(filename) - strlen(ext)], ext) == 0)
 
+/* Create proper HTTP Response Code */
+
+const char *translate_status_code(uint8_t ts_status_code)
+{
+    switch (ts_status_code){
+        case  TS_STATUS_CREATED:
+            return "201";
+        case TS_STATUS_DELETED:
+            return "204";
+        case TS_STATUS_VALID:
+            return "200";
+        case TS_STATUS_CHANGED:
+            return "204";
+        case TS_STATUS_CONTENT:
+            return "200";
+        case TS_STATUS_BAD_REQUEST:
+            return "400";
+        case TS_STATUS_REQUEST_INCOMPLETE:
+            return "400";
```

Comment:
We can save some memory by doing:

```suggestion
        case TS_STATUS_CONTENT:
        case TS_STATUS_VALID:
            return "200";
        case  TS_STATUS_CREATED:
            return "201";
        case TS_STATUS_CHANGED:
        case TS_STATUS_DELETED:
            return "204";
        case TS_STATUS_BAD_REQUEST:
        case TS_STATUS_REQUEST_INCOMPLETE:
            return "400";
```

Reply:
The compiler does it anyway :)

Reply:
Does it? Not sure how smart the compiler actually is... but in general it's not a big deal, was just an idea.

Reply:
I changed but the size of the imaged still was the same...looks better anyway :+1:

Reply:
Ok, good to know that the compiler got it optimized.